attic/jax_tricks/rescale.py

```diff
@@ -0,0 +1,18 @@
+import jax
+from jax import numpy as np
+import numpy as onp
+
+
+def rescale_coordinates(conf, indices, box, scales):
+    """Note: scales unused"""
+
+    mol_sizes = np.expand_dims(onp.bincount(indices), axis=1)
+    mol_centers = jax.ops.segment_sum(conf, indices) / mol_sizes
+
+    new_centers = mol_centers - box[2] * np.floor(np.expand_dims(mol_centers[..., 2], axis=-1) / box[2][2])
```

Comment:
Looks like an improvement -- will apply!

(Note that I moved this function from `jax_utils.py` to `attic/` without modification, since it looked broken / unused / doesn't reference the scales parameter)

Reply:
ah, sorry, I noticed that this was moved but not that it was moved to `attic/`. No need to modify in that case!